surfer/utils.py

```diff
@@ -23,6 +26,31 @@
     string_types = str
 
 
+class _MayaviThresholdPatch(object):
+    """Context to monkey-patch Mayavi 4.5
+
+    In Mayavi 4.5, filters seem to be missing a .point_data attribute that
+    Threshold accesses on initialization.
+    """
+    need_patch = LooseVersion(mayavi.__version__) <= LooseVersion('4.5.0')
```

Comment:
I'm a little confused with this is `<=`, shouldn't it be `==` based on the description above, which suggests a problem just on 4.5.0? That's the current version of mayavi so this will always be True, unless I'm missing something.